lib/Model/IMAPMessage.php

```diff
@@ -707,10 +714,57 @@ public function toDbMessage(int $mailboxId): \OCA\Mail\Db\Message {
 			in_array('junk', $flags, true)
 		);
 		$msg->setFlagNotjunk(in_array(Horde_Imap_Client::FLAG_NOTJUNK, $flags, true));
-		$msg->setFlagImportant(in_array('$important', $flags, true));
+		// @todo remove this as soon as possible @link https://github.com/nextcloud/mail/issues/25
+		$msg->setFlagImportant(in_array('$important', $flags, true) || in_array('$label1', $flags, true));
 		$msg->setFlagAttachments(false);
 		$msg->setFlagMdnsent(in_array(Horde_Imap_Client::FLAG_MDNSENT, $flags, true));
 
+		$allowed = [
+			Horde_Imap_Client::FLAG_SEEN,
+			Horde_Imap_Client::FLAG_ANSWERED,
+			Horde_Imap_Client::FLAG_FLAGGED,
+			Horde_Imap_Client::FLAG_DELETED,
+			Horde_Imap_Client::FLAG_DRAFT,
+			Horde_Imap_Client::FLAG_RECENT,
+			Horde_Imap_Client::FLAG_JUNK,
+			'junk',
+			'$junk',
+			'$seen',
```

Comment:
is this really an IMAP standard flag?

Reply:
no but it's gmail's $seen flag - do we want to make a tag out of it?